src/docker/Dockerfile

```diff
@@ -52,43 +52,31 @@ RUN apt-get update && apt-get install -y \
     automake autoconf
 
 # Git 2.0
-RUN apt-get install -y software-properties-common python-software-properties
+RUN apt-get update && apt-get install -y \
```

Comment:
@amihaiemil can't we just have somthing similar to https://github.com/yegor256/dockers/blob/master/xjm/Dockerfile? would be much shorter